### ISSUE TITLE
M> German  Version of Lerner & Nash Foster interview transcript 

### ISSUE THREAD

**Issue by BelovedAquila** [rchain/bounties#266]
**Goal:**  [[TRANSCRIPT][.srt][GERMAN] "Rchain Exclusive chat Lawrence Lerner & Nash Foster"](https://docs.google.com/document/d/1EWAapJoeZ1jKkD2rj1RABkc7Blsh5KnHcQSAOz399uQ/edi)
**Stored in:**  [/RChain/Members/Marketing-Community Dev/YouTube](https://drive.google.com/drive/folders/1zBZXvPMIC9yBneYI2c4x-xGwr-D1X1Ii) 

**Collaborator:** @BelovedAquila 
**Progress:** 95%
- [x] Download the English (original) transcript
- [x] Translate the content: 
- [x] @BelovedAquila : 45%
- [x] @Viraculous : 30%
- [x] @Ojimadu: 20%
- [ ]  @flowpoint 
- [x]  Check the spelling
- [x]  View and review the German transcript
- [ ]  Have it reviewed by others with deeper  understanding of German and English 
- [x] Upload the German (translated) transcript as immediate as possible 

**Suggested budget**: $3000
**Time:** 3 weeks maximum.

**Comment by lapin7** [comment]
@BelovedAquila please re-edit the first comment of this issue and take #262 as an example

**Comment by BelovedAquila** [comment]
@lapin7 alright, got it. Thank you

**Comment by BelovedAquila** [comment]
Don't know if it's ok this way @lapin7?. And please would it be necessary to store the progress made in the course of transcription  at  /drive/folders/1zBZXvPMIC9yBneYI2c4x-xGwr-D1X1Ii  or can it be stored after  full transcription?

**Comment by lapin7** [comment]
Please make a copy of https://drive.google.com/open?id=1nxkJw2Pu1YdeAdWCJz11pVrRKibqnk3NoIZRSO7KxvQ
And rename the file with German it.
Add the proper link to Goal .srt file in the first comment of this issue. (it's pointing now to Croat...)

**Comment by flowpoint** [comment]
i can review and check, i'm a nativly speaking german. beforehand, i need the collaborator role

**Comment by BelovedAquila** [comment]
@flowpoint alright. To become a collaborator you need send a collaboration request to @lapin7 via lapin7@gmail.com. Cheers!

**Comment by BelovedAquila** [comment]
@lapin7  the German transcript has been worked on and it has been uploaded (stored).

**Comment by flowpoint** [comment]
the german transcript should be revised, some spots arent quite right. i can, or should we go over it together?

**Comment by BelovedAquila** [comment]
@flowpoint alright, go ahead.

**Comment by flowpoint** [comment]
im on discord as flowpoint, havent found you

**Comment by BelovedAquila** [comment]
Alright, would find you friend. @flowpoint

**Comment by flowpoint** [comment]
i need access to the file to review it, i guess i have no googlegroups access @lapin7

**Comment by dckc** [comment]
@BelovedAquila would you please elaborate on how you earned roughly 45% of the $5000 budget in the past week or so? By way of example, see the [27 Jan breakdown of work][1] I gave to justify a large-ish budget for issue #185.

In #302, @flowpoint suggested that $5000 is not a suitable budget. At $40/hr, that's 125hrs. If we spread that over the suggested 3 weeks, it's 41hrs/week. 45% of that is 18.75hrs. Is that about how much time you spent?

[1]: https://github.com/rchain/Members/issues/185#issuecomment-360983331

**Comment by BelovedAquila** [comment]
@dckc similar budget was first set on other translation issues of the same transcript, if you check other translation budgets for the same transcript, you will find out that all of them as at the time of my writing has same budget, some even had more budget than was stated above. I borrowed a leaf from other budgets of similar issues I wonder why you had to single this one out.
I agree that $5000 is not a suitable budget, like I said above the budget was set based on other translation issues.
I don't speak German and while the translation above was a computer aided translation, it can't be said to be a verbatim copy of google translate. Significant work was done on getting it translated as the transcript itself was fraught with many errors. 
There might still be errors in the translated files, definitely, but what would be the essence of reviewing if I am pretty sure there are no errors?
I am open to your suggestions.

**Comment by dckc** [comment]
Since you agree $5000 is not a suitable budget, I removed it from the issue description.

I suggest you come up with a suitable budget.

I vaguely recall a suggestion of basing translation budgets on word counts, with reference to a U. N. norm of some sort. Perhaps that's a place to start.

And since you don't speak German, your role in this task is at most organizational, right? Perhaps 5% to 15% of the work?

**Comment by BelovedAquila** [comment]
@dckc you got me pondering on the virtue behind outrightly removing a budget on an issue which you're not participating on because you have edit access? That feels like censorship.

The budget on this translation isn't unique to this issue alone, have you only seen this issue? Friend, I understand what you are trying to point out on this, but please don't make it seem to be that you are  more solely concerned about this issue to the abandonment of other similar issues? Let's make a balanced judgment. 

Dckc writes:

> And since you don't speak German, your role in this task is at most organizational, right? Perhaps 5% to 15% of the work?

I don't quite agree because clearly, this goes beyond just "organizational". Work was done in translating, editing and making the sentences logical to be read. 
While I believe a native speaker is likely more accurate, being a native doesn't second to being an appropriate translator, and not being a native doesn't prevents an appropriate translation, it can't be a perfect transcript even if it should have been, because the English transcript given for this to be done was hectic with errors, which one needs return to the video to make necessary corrections where possible. 
This work is very much open to reviews and errors pointed out.

**Comment by dckc** [comment]
As I explained Feb 8 above, I am participating here in response to a comment by @flowpoint on #302. While I am not as interested in translation as other aspects of advancing RChain, I do happen to have [15 years of experience][15] building an international community of collaborators that produced [hundreds of translations](https://www.w3.org/Consortium/Translation/) of technical documents in dozens of languages; I also worked on [internationalizing HTML][2070] and other core Web technologies. So when the issue of rules for translations (#302) came up, I felt obliged to share what I know.

[15]: https://www.w3.org/blog/2010/06/thanks-for-a-great-15-years-at/
[2070]: https://tools.ietf.org/html/rfc2070

I don't understand what you mean by "editing and making the sentences logical to be read" if you don't speak German.

As to other issues, no, I have not seen any other specific complaints raised.

@flowpoint, @Ojimadu, once @BelovedAquila said "I agree that $5000 is not a suitable budget" I thought it would be helpful to update the issue summary to match. Is that censorship?

**Comment by flowpoint** [comment]
my stand on this is, i am a **validator.**
i proofread the documents and fixup the **few** spelling, punctuation, capitalization mistakes that **slipped through**, and point out **formulation** that can be improved to convey more meaning.
but in this case the accuracy is so far off, that a major revision is needed.
#302 translating to languages that you dont speak is not very productive even if it took you good effort.
the only requirement for translation is that you have **sufficient** understanding of **both** languages
i am unsure as to why @Ojimadu @Viraculous approved the document, as there are obvious mistakes still present.
a translation has not only to fullfill the rules of the given language, but also has to convey the **meaning** of the source.
for subtitles this is even more difficult as there are chronological constraints.
i guess, because the transcript was approved by you three, that you dont have the ability to improve it much further, if else let me know and do so.
i have no intentional bias against anyone, just the desired accurracy isnt reached.

@dckc i dont think it is censorship, but rather it just feels bad to be exploitable through someone else editing your messages

**Comment by lapin7** [comment]
Every line starts with a capital. Like:
**Hier etwas Hintergrund zu den beiden**:
This is very irritating to me. Maybe it needs a change.
And what about **zu den** >> **zum**

**Comment by dckc** [comment]
@flowpoint writes:
> in this case the accuracy is so far off, that a major revision is needed. ... i am unsure as to why @Ojimadu @Viraculous also approved the document, as there are obvious mistakes still present.

In what sense did they approve the document? This issue is still open. Has the document been posted to some widely visible location without a clear indication that it is still work-in-progress?

@flowpoint continues:
> @dckc i dont think it is censorship, but rather it just feels bad to be exploitable through someone else editing your messages

OK, reverted my edit to the suggested budget and just added a "_withdrawn Feb 9_" link. It's important that the issue summary doesn't mislead someone into thinking that $5000 is still the suggested budget when in fact noone is suggesting that budget any longer.

**Comment by flowpoint** [comment]
@dckc approved in my eyes, because the checkbox for spellcheck and review of the document is ticked

**Comment by dckc** [comment]
@ICA3DaR5 is this ready to close?

**Comment by ICA3DaR5** [comment]
There are two files in the directory.

1. https://drive.google.com/open?id=1oTZVkYEhhnZtyU4DmNiiT3N39R0IVpHI92vP8kaltFs
2. https://drive.google.com/open?id=1EWAapJoeZ1jKkD2rj1RABkc7Blsh5KnHcQSAOz399uQ

Which is the right one?

@lapin7 any insights?

Also, the files need to be uploaded as .srt files. Like [the english one](https://drive.google.com/open?id=1nl5ctTmA9FNK_uS6K2nN2m3gyzZJCIgY) in order to be ready for upload to YouTube.

**Comment by ICA3DaR5** [comment]
@flowpoint tells me that the right one is No 2. BUT it's not finished yet!

**Comment by lapin7** [comment]
@ICA3DaR5 please keep one file. You probably know best which one is the best.

**Comment by ICA3DaR5** [comment]
@lapin7 I am told by @flowpoint that [this](https://drive.google.com/open?id=1EWAapJoeZ1jKkD2rj1RABkc7Blsh5KnHcQSAOz399uQ) is the right file.

**Comment by lapin7** [comment]
I've corrected #266. I don't want to make that a habit, but I think this time it's necessary.
@flowpoint  did a good job on the German translation. B, O, V didn't participate in [that version](https://docs.google.com/document/d/1oTZVkYEhhnZtyU4DmNiiT3N39R0IVpHI92vP8kaltFs/edit).


Member | 86.50% | 21 | Voted percentages | 100.00% | BelovedAquila |   | 60.00% | lapin7
-- | -- | -- | -- | -- | -- | -- | -- | --
BelovedAquila | 6.00% | 5 | BelovedAquila 0.45 lapin7 -1.4 ICA3DaR5 0.4 Ojimadu 0.45 Viraculous 0.4 | 45% | 1 | BelovedAquila 0.45 | -140.00% | 1
lapin7 | 0.00% | 0 |   |   | 0 |   | 0.00% | 0
flowpoint | 76.00% | 5 | BelovedAquila 0.05 lapin7 3.5 ICA3DaR5 0.1 Ojimadu 0.05 Viraculous 0.1 | 5.00% | 1 | BelovedAquila 0.05 | 350.00% | 1
dckc | 0.00% | 0 |   |   | 0 |   | 0.00% | 0
ICA3DaR5 | 0.00% | 2 | ICA3DaR5 0.3 Ojimadu -0.3 | 0.00% | 0 |   | 0.00% | 0
Ojimadu | 2.00% | 5 | BelovedAquila 0.2 lapin7 -0.7 ICA3DaR5 0.2 Ojimadu 0.2 Viraculous 0.2 | 20.00% | 1 | BelovedAquila 0.2 | -70.00% | 1
Viraculous | 2.50% | 4 | BelovedAquila 0.3 lapin7 -0.8 Ojimadu 0.3 Viraculous 0.3 | 30.00% | 1 | BelovedAquila 0.3 | -80.00% | 1

**Comment by BelovedAquila** [comment]
@lapin7 I think what @flowpoint actually did was a review of the actual file we worked on, so I don't understand your bases of judgment on this please

**Comment by Ojimadu** [comment]
@lapin7 I don't think the reward you set is fair. I would have preferred @flowpoint doing this himself instead of doing it on his behalf.
 I am surprised to see that there are two translated files at the moment. 
This was work supposed to be collaborative but for @flowpoint to be working on another file without any prior disclosure here feels odd. He created the impression that he reviewed the original file while he was working on another, if he has/had reviewed the file and felt the errors corrected were so much he could have simply posted it here and requested the percentage be set to what he would see as fair to compensate for the work he did or He could have simply created another issue if he did not want to collaborate on this one stating reasons why he did. Up to this moment, he has not set a reward on this issue (which would have also informed mine). So setting the reward at this moment based only on the file he seemingly wants to work on alone (view only) is not fair as it negates the work done by others on this issue.

**Comment by flowpoint** [comment]
i was validator until yesterday, now i will contribute here.
i didnt disclose my file, because i didnt see my contribution beeing of mergeable quality then.
upon beeing asked where we are on this issue, i specified my file as the most up to date.
sry for missing the transparency.

**Comment by Viraculous** [comment]
This negates the rule of cooperation which is an intrinsic quality of the coop. Good job it is you have done @flowpoint but there would have been a better quality work if efforts on the new document was applied to the existing document. like @Ojimadu said, if you felt that the errors corrected were much and therefore deserving greater compensation, you could have indicated and then justice would be done to it or better still if you insist, there would have been a prior notice to the participants rather than assuming that you are working with existing document.

**Comment by Ojimadu** [comment]
Am taking your explanation with a "pinch of salt" as I don't think the reasons you provided are enough for the non-disclosure. I feel that being a German native you wanted to work on the issue alone. 
Am not comfortable continuing with this issue hence I am withdrawing myself from it. (*-*)

**Comment by BelovedAquila** [comment]
@ICA3DaR5 those are one of the issues one tends to face with our budget sheets, invariably the vote made on you is more of a impersonation by a bad actor. I believe @lapin7 or @dckc might be able  trace  out who perpetuated such act,sorry about that. 

And in respect to this issue and the not straightforward way it's turning around to be, ain't also comfortable progressing  with it this way,  have to unassign from it.

**Comment by lapin7** [comment]
I took a drastic measure. But I've investigated the work being done on the first translation initiated by Aldo (github name unknown). It was a Google translate. Then @BelovedAquila and @Ojimadu and @Viraculous step in. They didn't change a lot. Then @flowpoint saw this work. And to my opinion it was complete crap. So it was more practical for him to start a new file by his own and make a proper translation from there.

So for me it seems obvious that the time spent on translating English to German by NON-German speakers makes no sense at all.  Therefor only a small reward. And the main part has to go to @flowpoint .

Collaborators have to  understand that by delivering crap to RChain is by no way helping RChain forward. So if you don't know anything about a certain issue it's better to do nothing, because doing something would only harm the progress.

I don't want to abuse my power to change things. So this is probably the last time that I interfere in correcting obvious self interest. Anyway self interest above RChain interest is happening at all levels, so we have to keep up with these in-efficiencies. The message is to behave responsible and fair. And if you see that somebody doesn't behave like that then that has to be expressed and corrected openly.

But let's not get too busy with controlling each other. There may be some in-efficiencies but the overall goal to help RChain on the rails is the most important. Let's learn and go forward.
